src/main/java/com/develhope/spring/deals/responseStatus/OrderNotFoundException.java

```diff
@@ -0,0 +1,18 @@
+package com.develhope.spring.deals.responseStatus;
+
+import org.springframework.http.HttpStatus;
+
+import java.util.NoSuchElementException;
+
+public class OrderNotFoundException extends RuntimeException {
+    private HttpStatus httpStatus;
+
+    public OrderNotFoundException(String message, NoSuchElementException ex) {
+        super(message, ex);
+        this.httpStatus = HttpStatus.NOT_FOUND;

```

Comment:
cambiala in bad request. Dovremmo farle un po con tutte, perchè not found come diceva oggi non fa capire il problema visto che si usa per le pagine non trovate